multilep/test/testing/tests.log

```diff
@@ -1,42 +1,57 @@
+      _relIso_80X                                        mean:   0.2742 -->   0.2681     rms:   0.5991 -->   0.5962    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Run2018-PromptReco --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.1213 -->   0.1197     rms:   0.3497 -->   0.3490    
+      _relIso_80X                                        mean:   0.1555 -->   0.1494     rms:   0.3839 -->   0.3813    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Run2017-31March2017 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.7857 -->   0.7806     rms:   0.1427 -->   0.1440    
+      _relIso_80X                                        mean:   0.9415 -->   0.9301     rms:   0.0738 -->   0.0767    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Run2016-17Jul2018 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   1.6979 -->   1.6896     rms:   1.5892 -->   1.5889    
+      _relIso_80X                                        mean:   3.7779 -->   3.7619     rms:   4.4042 -->   4.3997    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Autumn18MiniAOD --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.1723 -->   0.1706     rms:   0.5232 -->   0.5215    
+      _relIso_80X                                        mean:   0.2647 -->   0.2601     rms:   0.6006 -->   0.5966    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Fall17MiniAODv2 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.0261 -->   0.0249     rms:   0.0780 -->   0.0762    
+      _relIso_80X                                        mean:   0.0761 -->   0.0697     rms:   0.1926 -->   0.1861    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Summer16MiniAODv3 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _leptonMvaTOP                                      mean:   0.8204 -->   0.8233     rms:   0.4736 -->   0.4705    
+      _leptonMvaTTH                                      mean:   0.8582 -->   0.8596     rms:   0.4314 -->   0.4301    
+      _leptonMvatZq                                      mean:   0.8088 -->   0.8139     rms:   0.4806 -->   0.4742    
```

Comment:
This is restoring the original values, which were broken in a former test run included in this pull request (https://github.com/GhentAnalysis/heavyNeutrino/commit/175de672638f563a8de6f912ddca8f7c92f4619f#diff-999fefd3e11bd7f25f9e57046ff7397dR53-R55).